src/linker/configuration.py

```diff
@@ -41,10 +41,14 @@ def get_step_directory(self, step_name: str) -> Path:
             step_dir = (
                 Path(os.path.realpath(__file__)).parent.parent.parent
                 / "steps"
-                / "pvs_like_case_study_sample_data"
+                / step_name
+                / "implementation"
```

Comment:
Nit: "steps" is plural and "implementation" is singular. Should we make them consistent, one way or the other?